platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -172,6 +172,9 @@ private void init() {
     }
   }
 
+  // objectName can remain null after the try block therefore objectName is not guaranteed null or
+  // not null in last ternary operation
+  @SuppressWarnings("squid:S2583")
```

Comment:
❗️ Same comment as earlier regarding `objectName` and `mBeanServer`: if `mBeanServer` is non-null, then `objectName` must be non-null as well.